package.json

```diff
@@ -31,5 +31,17 @@
   "homepage": "https://github.com/andpor/react-native-sqlite-storage",
   "peerDependencies": {
     "react-native": ">=0.14.0"
+  },
+  "dependencies": {
+    "rnpm": "1.5.2",
+    "xcode": "0.8.4"
+  },
+  "rnpm": {
+    "commands": {
+      "postlink": "node_modules/react-native-sqlite-storage/rnpm-postlink.js"
+    },
+    "ios": {
+      "project": "react-native-sqlite-storage/src/ios/SQLite.xcodeproj"
```

Comment:
this path does not seem to work, src/ios/SQLite.xcodeproj works better? Can you provide me with steps I can use to test your PR ?